src/Sylius/Component/Core/Taxation/Processor/OrderTaxesProcessor.php

```diff
@@ -68,7 +68,7 @@ public function process(OrderInterface $order)
             return;
         }
 
-        $zone = $this->getTaxZone($order->getShippingAddress());
+        $zone = $this->getTaxZone($order);
 
         if (null === $zone) {
             return;
```

Comment:
Is it possible not to have a tax zone for an order now? If it's not possible, we should throw an exception.